internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -187,8 +189,16 @@ func (c *scorecardCmd) run() (err error) {
 		log.Fatal(err)
 	}
 
+	podSecFlag := true
+	if c.podSecurity == "resticted" {
+		podSecFlag = true
+	} else if c.podSecurity == "legacy" {
+		podSecFlag = false
+	}
+
```

Comment:
A nit/typo:
```suggestion
	podSecFlag := true
	if c.podSecurity == "restricted" {
		podSecFlag = true
	} else if c.podSecurity == "legacy" {
		podSecFlag = false
	}

```

As an aside, since we are processing this as a boolean, would it be easier to just make this a boolean flag rather than a string flag? Maybe something like `--enable-psa`? This way we don't have to do any if statements like this where we are processing a string into a boolean.

Reply:
Somehow I missed this. No it should not be treated as a boolean. It should match what we're doing in run bundle. https://github.com/operator-framework/operator-sdk/pull/6062 we treat it as an enum to match what is in operator-framework/api.

Reply:
Would it be possible for the scorecard implementation to import and use the same logic that was created for `operator-sdk run bundle`? 

I'm fine to leave this as is if we are happy with it, but it would be nice to be using the same tooling/logic in all places we want this kind of functionality rather than having slightly different methods for each implementation.

Reply:
It might be possible, just seems weird. We would likely want to move the enum stuff out of run bundle into a common area so that it can be used by both sides: scorecard & run bundle.